api_v3/lib/types/filters/KalturaUserEntryFilter.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @package api
+ * @subpackage filters
+ */
+class KalturaUserEntryFilter extends KalturaUserEntryBaseFilter
+{
+	/**
+	 * @return baseObjectFilter
+	 */
+	protected function getCoreFilter()
+	{
+		// TODO: Implement getCoreFilter() method.
+		return new UserEntryFilter();
+	}
+
+	/**
+	 * @param KalturaFilterPager $pager
+	 * @param KalturaDetachedResponseProfile $responseProfile
+	 * @return KalturaListResponse
+	 */
+	public function getListResponse(KalturaFilterPager $pager, KalturaDetachedResponseProfile $responseProfile = null)
+	{
+
+		$response = new KalturaUserEntryListResponse();
+		if ( in_array(kCurrentContext::getCurrentSessionType(), array(kSessionBase::SESSION_TYPE_NONE,kSessionBase::SESSION_TYPE_WIDGET)) )
+		{
+			$response->totalCount = 0;
+			return $response;
+		}
+
+		$c = new Criteria();
+		$userEntryFilter = new UserEntryFilter();
```

Comment:
Let the $this->toObject instantiate the core filter for you, or call $this->getCoreFilter(), the used core filter could be extended by KalturaUserEntryFilter extending classes.